services/harmony/test/service-image-tags.ts

```diff
@@ -34,6 +34,10 @@ const serviceImages = {
   'podaac-l2-subsetter': 'sit',
   'query-cmr': 'latest',
   'sds-maskfill': 'latest',
+  'giovanni-adapter': 'latest',
+  'geoloco': 'latest',
+  'subset_band_name': 'latest',
```

Comment:
nitpick: all the other services use kabob case instead of snake case

Reply:
ah, good catch. Changed to kabob case.